scripts/move_to_scala_2.11.sh

```diff
@@ -11,11 +11,7 @@ then
     exit 1
 fi
 
-find . -name "pom.xml" -exec sed -e "s/2.10.6/2.11.12/g" \
-    -e "s/2.10/2.11/g" \
+find . -name "pom.xml" -exec sed -e "s/2.12.8/2.11.12/g" \
+    -e "s/2.12/2.11/g" \
```

Comment:
I don't think these files are being referenced in jenkins-test anymore. Do we want to run jenkins-test on scala 2.11 and 2.12?

Reply:
Chicken and egg unfortunately, Spark 2.4.x is required for Scala 2.12, and ADAM won't work on Scala 2.12 until this pull request